From a54773644858dff8e7543bb767f6fc4ca27ad7e5 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Sep 2024 22:36:52 +0800
Subject: [PATCH 25/35] [r38450] fmt-conversion: Remove reference to
 AV_PIX_FMT_XVMC

The pix fmt was removed from lavu on major bump to 59.

This fixes the build when vo xvmc is not enabled in configure.

As vo xvmc isn't automatically enabled, this commit should be
enough to avoid build breakage with newer FFmpeg for most people.
Even those that have the relevant dev packages installed.

More stuff related to XvMC (vo xvmc, parts in vd ffmpeg, etc) are
planned to be removed in follow up commits.

(From upstream SVN r38450)
---
 fmt-conversion.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fmt-conversion.c b/fmt-conversion.c
index a12c03e..d1d30af 100644
--- a/fmt-conversion.c
+++ b/fmt-conversion.c
@@ -138,7 +138,6 @@ static const struct {
     { IMGFMT_422P,       AV_PIX_FMT_YUVJ422P },
     { IMGFMT_444P,       AV_PIX_FMT_YUVJ444P },
     { IMGFMT_440P,       AV_PIX_FMT_YUVJ440P },
-    { IMGFMT_XVMC_IDCT_MPEG2, AV_PIX_FMT_XVMC },
     { 0,                      AV_PIX_FMT_NONE }
 };
 
-- 
2.46.0

